packages/components/pages/f-mfa/src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const SUBMIT_LOGIN_CHALLENGE_URL = 'api/account/mfa/login';
```

Comment:
I feel like this should perhaps be a prop that's provided by our consumers? At least that's what f-registration / f-checkout do - https://github.com/justeat/fozzie-components/blob/master/packages/components/pages/f-registration/src/components/Registration.vue#L258-L260